client/my-sites/plugins/plugin-action/plugin-action.jsx

```diff
@@ -27,7 +27,7 @@ module.exports = React.createClass( {
 			return (
 				<label
 					className="plugin-action__label"
-					ref="disabledInfoLabel"
+					ref={ this.props.disabledInfo ? 'disabledInfoLabel' : null }
```

Comment:
>I had to add this condition because otherwise tests were throwing errors.

I've seen this before, and I think it's a result of React being loaded twice as a consequence of using Mockery. Specifically, I think Enzyme may be loading a separate copy of React if importing both React and the Enzyme mount function. Importing in the `before` block (after Mockery has been initialized) is another way to solve the issue:

```diff
diff --git a/client/my-sites/plugins/plugin-action/plugin-action.jsx b/client/my-sites/plugins/plugin-action/plugin-action.jsx
index 2cb48d7..5248ead 100644
--- a/client/my-sites/plugins/plugin-action/plugin-action.jsx
+++ b/client/my-sites/plugins/plugin-action/plugin-action.jsx
@@ -27,7 +27,7 @@ module.exports = React.createClass( {
                        return (
                                <label
                                        className="plugin-action__label"
-                                       ref={ this.props.disabledInfo ? 'disabledInfoLabel' : null }
+                                       ref="disabledInfoLabel"
                                        onClick={ this.handleAction }
                                        htmlFor={ this.props.htmlFor }
                                        key="renderDisabledInfoLabel"
diff --git a/client/my-sites/plugins/plugin-action/test/index.jsx b/client/my-sites/plugins/plugin-action/test/index.jsx
index 85a4435..63c5f84 100644
--- a/client/my-sites/plugins/plugin-action/test/index.jsx
+++ b/client/my-sites/plugins/plugin-action/test/index.jsx
@@ -3,7 +3,7 @@
  */
 import { expect } from 'chai';
 import mockery from 'mockery';
-import { mount, shallow } from 'enzyme';
+import { shallow } from 'enzyme';
 import React from 'react';
 
 /**
@@ -14,12 +14,13 @@ import useFakeDom from 'test/helpers/use-fake-dom';
 import useMockery from 'test/helpers/use-mockery';
 
 describe( 'PluginAction', function() {
-       let PluginAction;
+       let mount, PluginAction;
 
        useFakeDom();
        useMockery();
 
        before( () => {
+               mount = require( 'enzyme' ).mount;
                mockery.registerMock( 'components/info-popover', EmptyComponent );
 
                PluginAction = require( '../plugin-action' );
```